CONTRIBUTING.md

```diff
@@ -1,26 +1,27 @@
 # Contributing Guide
 
 ---
-* [How to help](#how-to-help)
-  * [Find an issue](#find-an-issue)
-  * [When to open a pull request](#when-to-open-a-pull-request)
-  * [How to get your pull request reviewed fast](#how-to-get-your-pull-request-reviewed-fast)
-  * [The life of a pull request](#the-life-of-a-pull-request)
-* [Contribution Ladder](#contribution-ladder)
-  * [Contributor](#contributor)
-  * [How to become a contributor](#how-to-become-a-contributor)
-  * [Maintainer](#maintainer)
-  * [How to become a maintainer](#how-to-become-a-maintainer)
-  * [Admin](#admin)
-  * [How to become an admin](#admin)
-* [Developer Tasks](#developer-tasks)
-  * [Initial setup](#initial-setup)
-  * [Makefile explained](#makefile-explained)
-  * [Install mixins](#install-mixins)
-  * [Preview documentation](#preview-documentation)
-* [Code structure and practices](#code-structure-and-practices)
-  * [What is the general code layout?](#what-is-the-general-code-layout)
-  * [Logging](#logging)
+
+- [How to help](#how-to-help)
```

Comment:
It helps to not include unrelated changes in a pull request. Fixing a single bad link is fine to sneak in though, and it sounds like from your commit message that was the intent. Let's revert the changes to this file other than the missing `)` and this should be ready to merge. 🚀

Reply:
Sorry about that! I wanted to fix the missing `)` and I think my markdown plugin ran a format-on-save. I should have checked the diff before committing it! I'll get this fixed.